### PR TITLE
Exposure alert Initial UI Changes

### PR DESCRIPTION
**Pull request by Sam-gtri**
This is the initial pull request that incorporates the updates to the UI for the exposure alerts page

### CODE REVIEW

File: CoEpi/ui/alerts/AlertsViewModel.swift [hunk 1]
```diff
@@ -13,9 +13,8 @@ class AlertsViewModel {
     init(alertRepo: AlertRepo) {
         self.alertRepo = alertRepo
 
-        title = alertRepo.alerts
-            .map { AlertsViewModel.formatTitleLabel(count: $0.count) }
-            .startWith(AlertsViewModel.formatTitleLabel(count: 0))
+        title = alertRepo.alerts.map { _ in AlertsViewModel.formatTitleLabel() }
```
Comment (i-schuetz) [inline CoEpi/ui/alerts/AlertsViewModel.swift:16]: I'd remove this and set the translation directly in the view controller. The title doesn't depend on the alerts anymore.

File: CoEpi/ui/alerts/AlertsViewModel.swift [hunk 2]
```diff
@@ -39,22 +38,17 @@ class AlertsViewModel {
...
-        case 1: return L10n.Alerts.Count.one
-        default: return L10n.Alerts.Count.some(count)
-        }
-    }
+    private static func formatTitleLabel() -> String {
+        return L10n.Alerts.header    }
 }
 
 private extension FeverSeverity {
     // TODO localize
```
Comment (i-schuetz) [inline CoEpi/ui/alerts/AlertsViewModel.swift:46]: Can be removed, since you solved it.

File: CoEpi/ui/alerts/AlertsViewModel.swift [hunk 3]
```diff
@@ -63,10 +57,10 @@ private extension CoughSeverity {
...
```
Comment (i-schuetz) [inline CoEpi/ui/alerts/AlertsViewModel.swift:57]: Can be removed.

**Comment by Sam-gtri** [comment]
Ill work on these changes today

**Comment by Sam-gtri** [comment]
@i-schuetz I just committed the changes - let me know if you think this is ready for merging

**Comment by i-schuetz** [comment]
Approved! Please resolve the conflicts and merge.

**Pull request merged**